# &#x1F4DC; Lecture 1

Small-scale multipath fading is more relevant to the design of reliable and efficient communication systems - The focus of the book (Fundamental of WIRELESS COMMUNICATION)

## &#x274D; Communication (Simplistic View)

<img src=images/lecture-1_Simplistic-View.png width='30%' height='30%' ></img>

Transmitted sinusoid: $x(t) = \cos(2 \pi \mathcal{f} t) $ where : 
- $ \mathcal{f} $ is frequency $\approx 1 \to 4 $ Ghz
- $ t $ is time

$$
\begin{equation}
\text{Rx: } E(f, t, r, \theta, \psi) = \frac{1}{r} \cdot {\color{yellow}\alpha}_s(\theta, \psi, f) \cdot \cos\left( 2 \pi f \left( t - {\color{orange}\frac{r}{c}} \right) \right)
\end{equation}
$$

- $ {\color{yellow}\alpha} $: antenna losses
- $ {\color{orange}\frac{r}{c}} $: delay due to signal


**Intuition why $ E \propto \frac{1}{r} $:**

- Tx in 3-D space.
- Power preserved in surface of sphere: (radius $ r $, center is antenna).
- $ \text{Fixed Total Power / Area} $ → Rx power $ \propto \frac{1}{4 \pi r^2} $ → $ \propto \frac{1}{r^2} $.

## &#x274D; Consider Movement

<img src=images/lecture-1_consider_movement.png width='50%' height='50%'> </img>

The handwritten note contains the following derivation and explanations:

1. **Equation for $ E(f, t, r(t)) $**:
   $
   E(f, t, r(t)) = \frac{\alpha_s}{r(t)} \cos \left( 2 \pi f \left( t - \frac{r(t)}{c} \right) \right)
   $
   - Where $ r $ is now a function of time, $ r(t) $.

2. **Simplification**:
   $
   = \frac{\alpha_s}{r_0 + v \cdot t} \cos \left( 2 \pi f \left( t - \frac{r_0}{c} - \frac{v \cdot t}{c} \right) \right)
   $
   - Here, $ r(t) $ is expanded as $ r_0 + v \cdot t $, where $ r_0 $ is the initial distance and $ v $ is the relative velocity.

3. **Further Simplification**:
   $
   = \frac{\alpha_s}{r_0 + v \cdot t} \cos \left( 2 \pi f \left[ t \left( 1 - \frac{v}{c} \right) - \frac{r_0}{c} \right] \right)
   $

4. **Concept of Effective Frequency**:
   - Effective frequency change:
     $
     f t \rightarrow f \left( 1 - \frac{v}{c} \right) \cdot t
     $
   - This implies that the frequency $ f $ is effectively reduced to $ f \left( 1 - \frac{v}{c} \right) $.

5. **Frequency Reduction**:
   - The effective frequency becomes:
     $
     f \rightarrow f \left( 1 - \frac{v}{c} \right)
     $
   - This is related to the **Doppler shift**.

6. **Doppler Shift**:
   - The Doppler shift $ D $ is given by:
     $
     D = -\frac{f v}{c}
     $

7. **Final Expression**:
   $
   E(f, t, r(t)) = \frac{\alpha_s}{r_0 + v \cdot t} \cos \left( 2 \pi f \left( t \left( 1 - \frac{v}{c} \right) - \frac{r_0}{c} \right) \right)
   $
   - Note: The time is now modified due to the relative motion.
   - Note: $\frac{\alpha_s}{r_0 + v \cdot t}$ Not LTI now.

This derivation shows how the received electric field $ E $ is affected by the relative velocity $ v $ between the transmitter and receiver, resulting in a Doppler shift in frequency. The frequency observed at the receiver is reduced by a factor of $ \left( 1 - \frac{v}{c} \right) $ due to the relative motion, a characteristic effect of the Doppler phenomenon.

## &#x274D; Fixed Tx, Fixed Rx, Perfectly Reflecting Wall

<img src=images/lecture-1_fixed-TXRX_reflecting_wall.png width='50%' height='50%'> </img>

A diagram shows a transmitter (Tx), receiver (Rx), and a perfectly reflecting wall with two paths: one direct and one reflected.

1. **Field Equation**:
   - $ E_r(f, t) = \frac{\alpha}{r} \cos \left( 2 \pi f \left( t - \frac{r}{c} \right) \right) - \frac{\alpha}{2d - r} \cos \left( 2 \pi f \left( t - \frac{2d - r}{c} \right) \right) $
   - Here, $ r $ is the distance of the direct path, and $ 2d - r $ is the distance of the reflected path.

2. **Ray Tracing Method**:
   - Use ray tracing to consider the **dominant (main) paths**.

3. **Superposition**:
   - The received field is the **superposition (addition) of two sinusoids with different phases**.
   - This can cause a **possible effect of cancellation**.

4. **Phase Shift**:
   
   - Phase difference: 
     $
     \Delta \phi = \phi_2 + {\color{orange} \pi } - \phi_1
     $
       - The $ {\color{orange} \pi } $ phase shift occurs in the electric field of an EM wave when reflected from an optically denser medium.
       - Essentially **cancelling each other out close to the wall**.


     $
     \Delta \phi = 2 \pi f \frac{ 2d - r}{c} + {\color{orange} \pi } - 2  \pi f \frac{r}{c}
     $

To simplify the expression: $\Delta \phi = 2 \pi f \frac{2d - r}{c} + \pi - 2 \pi f \frac{r}{c}$

let’s combine terms:

1. Distribute $ 2 \pi f $ in the first term: $\Delta \phi = \frac{2 \pi f (2d - r)}{c} + \pi - \frac{2 \pi f r}{c}$

2. Separate the terms involving $ r $: $\Delta \phi = \frac{2 \pi f \cdot 2d}{c} - \frac{2 \pi f r}{c} + \pi - \frac{2 \pi f r}{c}$

3. Combine like terms involving $ r $: $\Delta \phi = \frac{4 \pi f d}{c} - \frac{4 \pi f r}{c} + \pi$

4. Factor out $ \frac{4 \pi f}{c} $ from the terms involving $ d $ and $ r $: $\Delta \phi = \frac{4 \pi f}{c} (d - r) + \pi$

##### Final Answer $\boxed{\Delta \phi = \frac{4 \pi f}{c} (d - r) + \pi}$

This is the simplified form of the original expression.

---

This note explains how the electric field at the receiver $ E_r $ is the result of two paths: the direct path and the reflected path. The reflected path introduces a phase shift, which, due to superposition, may lead to constructive or destructive interference, potentially canceling out the signal near the reflecting wall.

## &#x274D; **Coherence Bandwidth:**

- Definition: The change in frequency $ f $ such that the channel (i.e. magnitude of received signal) remains relatively the same.

1. **Phase Shift Due to Frequency Change**:
   $
   \Delta \phi = \frac{4 \pi f }{c} (d - r) + \pi
   $
   - The goal is to find a change in $ f $ so that the phase shift $ \Delta \phi \approx \frac{\pi}{2} $.

2. **Approximation for Small Phase Shift**:
   - Rewrite the phase condition as:

     - 
     $
     \frac{4 \pi f}{c} ( d - r ) \approx \frac{\pi}{2}
     $

     - 
     $
     2 \pi f \left( \frac{2d - r \; - r}{c} \right) \approx \frac{\pi}{2}
     $
     to help us later.
     - This is simplified to get an approximate frequency change.

3. **Delay Spread**:
   - The quantity $ \frac{2d - r - r}{c} $ is defined as the **delay spread $ T_d $**, which is the **difference in delay** between paths.
   
4. **Relationship to Coherence Bandwidth**:
   - Setting $ 2 \pi f T_d \approx \frac{\pi}{2} $, solve for $ f $:
     $
     f \approx \frac{1}{4 T_d} \Rightarrow W_c = \frac{1}{4 T_d}
     $
   - Alternatively, if a substantial phase shift (like $ \pi $) is considered:
     $
     W_c = \frac{1}{2 T_d}
     $
   
5. **Conclusion**:
   - Coherence bandwidth $ W_c $ is approximately $ \frac{1}{T_d} $, as noted in references like Wikipedia.

---

This note derives coherence bandwidth $ W_c $ as inversely proportional to the delay spread $ T_d $, with different constants (e.g., $ \frac{1}{4 T_d} $ or $ \frac{1}{2 T_d} $) depending on the chosen phase shift threshold.

In [3]:
using Unitful: μs
using Unitful
using Markdown
# using LaTeXStrings     # Import LaTeXStrings to use L"..." for LaTeX-style formatting

# Given delay spread
T_d = 1.9μs         # Define T_d in microseconds

# Coherence angular frequency (in rad/μs)
W_c = 2 * π / T_d

# Convert to coherence bandwidth in MHz and round to 2 decimal places with unit type specified
B_c = round(typeof(Unitful.u"1.0MHz"), uconvert(Unitful.MHz, W_c / (2 * π)), digits=2)

println("Given delay spread (T_d): $T_d ")
println("Coherence angular frequency (W_c): $W_c")
println("Coherence bandwidth (B_c): $B_c")

Given delay spread (T_d): 1.9 μs 
Coherence angular frequency (W_c): 3.306939635357677 μs⁻¹
Coherence bandwidth (B_c): 0.53 MHz


The result  $W_c = 3.31 \, \mu s^{-1}$  obtained here is in units of inverse seconds $(s^{-1})$ or radians per microsecond $(rad/\mu s)$, which is the ***coherence angular frequency*** in radians per second if converted properly.

The handwritten note discusses **coherence distance** in the context of electromagnetic waves and phase shift.

---

## &#x274D; Coherence Distance

- **Definition**: The coherence distance $ \Delta x_c $ is the distance over which the phase shift remains relatively constant, meaning the channel characteristics (like phase) stay consistent.

1. **Phase Shift Condition**:
   $
   \Delta \phi \approx 2 \pi f \left( \frac{2d - r}{c} - \frac{r}{c} \right) = \frac{2 \pi f}{c} (2d - 2r) = \frac{4 \pi f}{c} (d - r)
   $
   - This phase shift is influenced by the distance $ d - r $, and the goal is to find the change $r$ in distance that results in a phase shift of approximately $ \frac{\pi}{2} $.

2. **Calculating Coherence Distance**:
   - Setting $ \Delta \phi = \frac{\pi}{2} $:
     $
     \frac{4 \pi f}{c} \times \Delta x \approx \frac{\pi}{2}
     $
   - Solving for $ \Delta x $:
     $
     \Delta x_c \approx \frac{c}{8f} \approx \frac{\lambda}{8}
     $
   - This gives the coherence distance $ \Delta x_c $ as approximately $ \frac{\lambda}{8} $, where $ \lambda $ is the wavelength.

3. **Alternative Calculation with Substantial Phase Shift**:
   - If a larger phase shift (e.g., $ \pi $) is chosen:
     $
     \Delta x_c \approx \frac{\lambda}{4}
     $
   - Depending on the convention, coherence distance could be approximated by either $ \frac{\lambda}{8} $ or $ \frac{\lambda}{4} $.

### Summary
- **Coherence distance $ \Delta x_c $** indicates the spatial range over which the phase remains stable.
- **Result**: $ \Delta x_c \approx \frac{\lambda}{8} $ or $ \frac{\lambda}{4} $, depending on the phase shift threshold used (e.g., $ \frac{\pi}{2} $ or $ \pi $).

This coherence distance defines the range over which the received signal characteristics are likely to be stable.

In [4]:
using Unitful: μm

# Define wavelength (λ) in micrometers (or choose an appropriate unit)
λ = 500μm  # Example wavelength, can be adjusted as needed

# Coherence distance calculations
Δx_c_λ_8 = λ / 8        # Approximation: Δx_c ≈ λ/8
Δx_c_λ_4 = λ / 4        # Approximation: Δx_c ≈ λ/4
Δx_c_λ_pi = λ / π       # Approximation: Δx_c ≈ λ/π

# Display results
println("Given wavelength (λ): $λ")
println("Coherence distance (Δx_c ≈ λ/8): $Δx_c_λ_8")
println("Coherence distance (Δx_c ≈ λ/4): $Δx_c_λ_4")
println("Coherence distance (Δx_c ≈ λ/π): $Δx_c_λ_pi")

Given wavelength (λ): 500 μm
Coherence distance (Δx_c ≈ λ/8): 62.5 μm
Coherence distance (Δx_c ≈ λ/4): 125.0 μm
Coherence distance (Δx_c ≈ λ/π): 159.15494309189535 μm


$
E_r(f, t) = \frac{\alpha}{r_1(t)} \cos 2 \pi f \left[ t - \frac{r(t)}{c} \right]  + \frac{\alpha}{ r_2(t)} \cos  \left[ 2 \pi f \left[ t - \frac{2d - r(t)}{c} \right] + \pi \right]
$

---
The handwritten note provides an analysis of **multipath fading** in a channel where a moving antenna (receiver or transmitter) experiences constructive and destructive interference due to **multiple paths** with different lengths. Here is a summary of the main points and equations presented:

## &#x274D; Reflecting wall, moving Antenna

<img src=images/lecture-1_reflecting_wall-moving_antenna.png width='70%' height='70%' > </img>

### Diagram and Setup

1. **Diagram and Variables**:
    - There is a **moving antenna** at a velocity $ v $, with a **distance $ r(t) = r_0 + v \cdot t $** between the transmitter and receiver.
    - The total distance between the transmitter and reflecting wall is $ d $.
    - The distance $ d - r $ represents the difference between the total distance $ d $ and $ r(t) $.
   - **Text**:

- **Channel strength changes as you move through constructive & destructive interference ("multipath fading").**

**Equation for $ E_r(f, t) $**:

$
\begin{align}
    E_r(f, t) &= \frac{\alpha}{r(t)} \cos \left[ 2 \pi f \left( t - \frac{r(t)}{c} \right) \right] + \frac{\alpha}{2d - r(t)} \cos \left( 2 \pi f \left( t - \frac{2d - r(t)}{c} \right) + \pi \right) \\
    &= \frac{\alpha}{r_0 + v \cdot t} \cos \left( 2 \pi f \left( t - \frac{v \cdot t}{c} - \frac{r_0}{c} \right) \right) + \frac{\alpha}{2d - r_0 - v \cdot t} \cos \left( 2 \pi f \left( t - \frac{2d - r_0 - v \cdot t}{c} \right) + \pi \right) \\
    &= \frac{\alpha}{r_0 + v \cdot t} \cos \left( 2 \pi f \left( t \left( 1 - \frac{v}{c} \right) - \frac{r_0}{c} \right) \right) - \frac{\alpha}{2d - r_0 - v \cdot t} \cos \left( 2 \pi f \left( t \left( 1 + \frac{v}{c} \right) - \frac{2d - r_0}{c} \right) \right)
\end{align}
$

   - Annotation: "π (turn here)" and "because reflection."

---

This transcription provides the exact handwritten equations and text without additional interpretation.


### Channel Characteristics and Multipath Fading

- **Channel Strength Changes**: The received signal strength changes as you move through **constructive and destructive interference** from multiple paths. This phenomenon is known as **multipath fading**.

### Expression for the Received Signal $ E_r(f, t) $

1. **Initial Received Signal Model**:
   $
   E_r(f, t) = \frac{\alpha}{r_1(t)} \cos \left( 2 \pi f \left( t - \frac{r(t)}{c} \right) \right) + \frac{\alpha}{2d - r(t)} \cos \left( 2 \pi f \left( t - \frac{2d - r(t)}{c} \right) + \pi \right)
   $
   - Here, the received signal $ E_r(f, t) $ consists of two components:
     - The first term represents the **direct path** with distance $ r(t) $.
     - The second term represents the **reflected path** from a wall with total distance $ 2d - r(t) $, and includes an additional phase shift of $ \pi $ (often due to reflection).

2. **Simplified Form**:
   - After simplifying the expression by substituting $ r(t) = r_0 + v \cdot t $, the equations are rewritten to show the Doppler effect due to the movement of the antenna.

### Doppler Effect and Frequency Shift

- As the antenna moves, the relative distance between the transmitter and receiver changes, which causes a **Doppler shift** in the frequency of the received signal.
- This Doppler shift results in **phase changes** over time, which affects the constructive and destructive interference pattern in the received signal.

### Key Points

- **Multipath Interference**: The constructive and destructive interference caused by the varying distances of the direct and reflected paths leads to multipath fading.
- **Phase Shift Due to Reflection**: The phase shift of $ \pi $ in the reflected signal results in different interference patterns, depending on the relative phases of the direct and reflected signals.

This setup and the resulting equations model how a moving antenna in a multipath environment experiences varying signal strength due to the interaction between direct and reflected paths, which is fundamental in understanding **multipath fading** in wireless communication.

---
# References

---

### Received Rlectric Field $ E $ 

$$
E(f, t, r, \theta, \psi) = \frac{1}{r} \cdot \alpha_s(\theta, \psi, f) \cdot \cos\left( 2 \pi f \left( t - \frac{r}{c} \right) \right)
$$

This equation represents the received electric field $ E $ from a signal source at a distance $ r $ from the receiver. The formula takes into account several factors including distance, angle of arrival, frequency, and signal delay due to propagation time.

Breaking down each term:

- **$ E(f, t, r, \theta, \psi) $**: The electric field strength at the receiver as a function of frequency $ f $, time $ t $, distance $ r $, and angles $ \theta $ and $ \psi $ (likely representing azimuth and elevation or other directional parameters).
  
- **$ \frac{1}{r} $**: This represents the attenuation due to the distance $ r $ from the source, assuming spherical spreading where the field strength decreases as $ 1/r $ with distance.

- **$ \alpha_s(\theta, \psi, f) $**: This factor represents the antenna gain pattern and signal strength at the receiver. It depends on the direction (angles $ \theta $ and $ \psi $) and frequency $ f $, capturing how the antenna gain varies with direction and frequency.

- **$ \cos\left( 2 \pi f \left( t - \frac{r}{c} \right) \right) $**: This term describes the oscillatory nature of the electromagnetic wave, where:
  - $ f $ is the frequency of the signal,
  - $ t $ is the time variable,
  - $ \frac{r}{c} $ is the delay due to the finite speed of light $ c $. 

The term $ t - \frac{r}{c} $ accounts for the time it takes for the signal to travel the distance $ r $, introducing a phase delay in the received wave.

### Interpretation
This expression models the electric field of a received signal from a distant transmitter, factoring in attenuation over distance, the directional characteristics of the antenna, and the time delay due to signal travel. It's a standard way to describe the received signal in scenarios involving free-space propagation with directional antennas, commonly used in fields like wireless communication and radar.



---

### $ \alpha $ a composite factor

In the context of the link budget equation or received power calculation, $ \alpha $ is often a composite factor that incorporates various parameters affecting the transmission and reception of electromagnetic waves. Let's break down what $ \alpha $ might typically include and how it can be expressed:

### 1. **Antenna Gains** ($ G_{\text{Tx}} $ and $ G_{\text{Rx}} $)
   - The gains of the transmitting and receiving antennas are crucial factors in determining the effective power transfer. Antenna gain describes how effectively an antenna can radiate or receive power in a particular direction compared to an isotropic radiator (an idealized antenna that radiates equally in all directions).
   - If both the transmitting and receiving antennas have directional gains, $ \alpha $ might include these terms as:
     $
     \alpha = G_{\text{Tx}} G_{\text{Rx}}
     $
   - $ G_{\text{Tx}} $ is the gain of the transmitting antenna, and $ G_{\text{Rx}} $ is the gain of the receiving antenna.

### 2. **Path Loss Factor** (including **wavelength** or **frequency** dependency)
   - In free-space propagation, the received power is also affected by the wavelength $ \lambda $ (or equivalently, the frequency $ f $). The **Friis transmission formula** in free space includes a term that is proportional to $ \left( \frac{\lambda}{4 \pi r} \right)^2 $.
   - This factor is often included in $ \alpha $ as:
     $
     \alpha = \frac{\lambda^2}{(4 \pi)^2} G_{\text{Tx}} G_{\text{Rx}}
     $
   - Since $ \lambda = \frac{c}{f} $, where $ c $ is the speed of light, this factor implies that $ \alpha $ will depend on the frequency, which is essential in calculating the received power at different operating frequencies.

### 3. **Antenna Losses** ( $ L $ or **Efficiency $ \eta $**)
   - Real antennas are not perfectly efficient, meaning they have losses due to material properties, impedance mismatches, and other inefficiencies. These losses are often represented by a factor $ L $, which reduces the effective gain of each antenna.
   - In terms of efficiency $ \eta $, where $ \eta $ is the fraction of power actually radiated or received:
     $
     \alpha = \eta_{\text{Tx}} G_{\text{Tx}} \cdot \eta_{\text{Rx}} G_{\text{Rx}}
     $
   - Alternatively, if $ L $ represents the combined losses in the system, $ \alpha $ might be expressed as:
     $
     \alpha = \frac{G_{\text{Tx}} G_{\text{Rx}}}{L}
     $

### 4. **Environmental and Propagation Losses**
   - Depending on the propagation environment, $ \alpha $ may also account for additional losses due to obstacles, reflections, refraction, and scattering. These losses can vary significantly in non-line-of-sight (NLOS) conditions or in environments with high multipath effects.
   - In complex models, $ \alpha $ might be adjusted with empirical factors or measured data to reflect these environmental impacts. For instance, urban or indoor environments might include an extra path loss exponent, adjusting $ \alpha $ accordingly.

### Summary
Putting it all together, $ \alpha $ is typically a combined factor that could look like:

$
\alpha = \frac{\lambda^2}{(4 \pi)^2} \frac{G_{\text{Tx}} G_{\text{Rx}}}{L}
$

or, incorporating efficiencies:

$
\alpha = \eta_{\text{Tx}} \eta_{\text{Rx}} \frac{\lambda^2}{(4 \pi)^2} G_{\text{Tx}} G_{\text{Rx}}
$

In this way, $ \alpha $ encapsulates the **gains, losses, efficiency, wavelength dependency,** and potentially **environmental factors** affecting the received power. It’s a useful simplification in link budget calculations to account for all these parameters in a single term.